src/api/server.js

```diff
@@ -0,0 +1,84 @@
+import { createServer, Model, RestSerializer } from 'miragejs';
+import faker from 'faker';
+
+faker.seed(28);
+
+export function setupMockServer() {
+    createServer({
+        serializers: {
+            application: RestSerializer,
+        },
+
+        models: {
+            video: Model,
+            playList: Model,
+            likedVideo: Model,
+        },
+
+        routes() {
+            this.namespace = 'api';
+            this.timing = 200;
+            this.resource('videos');
+            this.resource('playLists');
+            this.resource('likedVideos');
```

Comment:
Aviod using capital letters in api i.e you name them as e.g playlists and likedvideos

Reply:
I'll take care of this while making the backend